README.md

```diff
@@ -369,7 +369,7 @@ You can also specify a UUID to use with the credentials, which will be used to
 identify the credentials from within the job config. This is necessary when setting
 credentials for use with the [git plugin](http://docs.openstack.org/infra/jenkins-job-builder/scm.html#scm.git), for example.
 
-You can either manually generate a UUID from a site like https://www.uuidgenerator.net,
+You can either manually generate a UUID from a site like [UUIDTools.com](https://www.uuidtools.com/generate/v4),
```

Comment:
Alternative is `python -c 'import uuid ; print(uuid.uuid4())'`. Ruby appears to need a gem.

Reply:
That will certainly work. There is also a uuid cli (https://linux.die.net/man/1/uuid). Just thought I'd update the link to an open source version. Clicking a link will be faster for some people.